litmus-portal/frontend/src/views/Community/GeoMap/styles.ts

```diff
@@ -29,34 +29,35 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   cityMapComposableMap: {
-    width: '54rem',
-    height: '40rem',
+    width: '45vw',
```

Comment:
refrain from using `vw` unnecessarily...use `%` or `rem` instead

Reply:
it's not working using either because plotly and the map is not reacting to changes in rem or %, only vw works here, also discussed the same with @vanshBhatia-A4k9

Reply:
Make "Removing plotly and using litmus-ui graphs" an action item for the next release please.